source/build/Android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Apr 04 21:15:54 PDT 2019
+#Mon Dec 02 14:37:52 PST 2019
```

Comment:
nit: this will change whenever someone has to update the Android portion of the xplat sdk. Should this file be added to the .gitignore file? Or would this file not be properly generated if we open the Android Studio test project?

Reply:
I don't know enough about Android Studio to know for sure.
I feel the safest assumption today is to just assume the initial creator knew what they were doing.
...
According to a casual search, the first, highest-rated responses to multiple copies of this question seem to indicate that this file should be included, and not ignored.